### ISSUE TITLE
2000 RA Callout not sounding

### ISSUE THREAD

**Issue by WinLux-art** [flybywiresim/aircraft#9381]
### Aircraft Version

Development

### Build info

```json
{
    "built": "2024-11-02T19:03:13+00:00",
    "ref": "refs/tags/v0.12.1",
    "sha": "2776100c286d3723826c62df36dc48207657d937",
    "actor": "Benjozork",
    "event_name": "manual",
    "pretty_release_name": "stable/v0.12.1",
    "version": "a380x-v0.12.1-rel.2776100"
}
```

### Describe the bug

When the RA 2000 is turned on in the flypad (toggle switch is blue), it does not sound dispite 2000 being shown on the bottom of the artificial horizon.

### Expected behavior

The excpected behavior is that when the aircraft reaches 2000 ft agl and 2000 agl is shown on the AH, the GPWS is supposed to callout "2000".

### Steps to reproduce

1. I went to the fly pad and to automatic callouts
2. I toggled (turned blue) the 2000 callout.
3. I descended to 2000 ft agl and below.

### Discord Username (optional)

rithr

**Comment by alepouna** [comment]
Reproduced

**Comment by tracernz** [comment]
Known issue that we forgot about. Should be addressed in the not too distant future.

Note: the callouts come from the FWS on Airbus aircraft; not the GPWS/TAWS.

**Comment by flogross89** [comment]
It seems that `new_2000 ` is missing from the Wwise project

**Comment by tracernz** [comment]
Yep, not sure how far Boris got with addressing that. It is in the A32NX.
